docs/auth/tokens.rst

```diff
@@ -3,77 +3,195 @@
+
+    {
+        "created": "2018-09-06T09:08:43.762697Z",
+        "id": "3a6b94b5-d20e-40bd-a7cc-521f5c79fab3",
+        "last_used": null,
+        "name": "my new token",
+        "perm_manage_tokens": false,
+        "token": "4pnk7u-NHvrEkFzrhFDRTjGFyX_S"
+    }
+
+Field details:
+
+``created``
+    :Access mode: read-only
+    :Type: timestamp
+
+    Timestamp of token creation, in ISO 8601 format (e.g.
+    ``2018-09-06T09:08:43.762697Z``).
+
+``id``
+    :Access mode: read-only
+    :Type: UUID
 
-The server will respond with a list of token objects, each containing
-timestamps of when the token was created and last used (or ``null``; note the
-``Z`` indicating UTC timezone), and a UUID to identify that token. Furthermore,
-each token can carry a name that has no operational meaning (it is meant for
-user reference only). Certain API operations will automatically populate the
-``name`` field with suitable values such as "login" or "dyndns".
+    Token ID, used for identification only (e.g. when deleting a token). This
+    is *not* the token value.
 
-::
+``last_used``
+    :Access mode: read-only
+    :Type: timestamp (nullable)
 
-    [
-        {
-            "created": "2018-09-06T07:05:54.080564Z",
-            "id": "3159e485-5499-46c0-ae2b-aeb84d627a8e",
-            "last_used": "2019-04-29T18:01:09.894594Z",
-            "name": "login",
-        },
-        {
-            "created": "2018-09-06T08:53:26.428396Z",
-            "id": "76d6e39d-65bc-4ab2-a1b7-6e94eee0a534",
-            "last_used": null,
-            "name": ""
-        }
-    ]
+    Timestamp of when the token was last successfully authenticated, or
+    ``null`` if the token has never been used.
 
-You can also retrieve an individual token by appending ``{id}/`` to the URL,
-for example in order to look up a token's name or creation timestamp.
+    In most cases, this corresponds to the last time when an API operation
+    was performed using this token.  However, if the operation was not
+    executed because it was found that the token did not have sufficient
+    permission, this field will still be updated.
 
+``name``
+    :Access mode: read, write
+    :Type: string
 
-Create Additional Tokens
-````````````````````````
+    Token name.  It is meant for user reference only and carries no
+    operational meaning.  If omitted, the empty string is assumed.
 
-To create another token using the token management interface, issue a
-``POST`` request to the same endpoint::
+    Certain API operations will automatically populate the ``name`` field with
+    suitable values such as "login" or "dyndns".
+
+``perm_manage_tokens``
+    :Access mode: read, write
+    :Type: boolean
+
+    Permission to manage tokens (this one and also all others).  A token which
+    does not have this flag set cannot access the ``auth/tokens/`` endpoints.
+
+``token``
+    :Access mode: read-once
+    :Type: string (28 characters)
```

Comment:
Let's be not too specific with details here - this value could change in the future

Reply:
done